src/PIL/Image.py

```diff
@@ -1318,6 +1319,27 @@ def getexif(self):
 
         return self._exif
 
+    def getxmp(self):
+        """
+        Returns an object containing the xmp tags for a given image.
+        :returns: XMP tags in an object.
```

Comment:
```suggestion
        Returns a dictionary containing the xmp tags for a given image.
        :returns: XMP tags in a dictionary.
```
Suggestion for clarity.